### PR TITLE
[NDK] mmtypes.h: Fix a copypasta in a comment

### PR DESCRIPTION
**Pull request by SergeGautherie**
## Purpose

Trivial documentation improvement.

## Proposed changes

- s/NtCreateSection/NtMapViewOfSection/.

### CODE REVIEW

File: sdk/include/ndk/mmtypes.h [hunk 1]
```diff
@@ -208,7 +212,6 @@ typedef enum _MI_VAD_TYPE
 } MI_VAD_TYPE, *PMI_VAD_TYPE;
 
 #ifdef NTOS_MODE_USER
-
```
Comment (HBelusca) [inline sdk/include/ndk/mmtypes.h:211]: Why this change? (removing a newline here, but you added one in a similar case above)

File: sdk/include/ndk/mmtypes.h [hunk 2]
```diff
@@ -100,11 +101,13 @@ extern "C" {
...
 #define SEC_LARGE_PAGES                                     0x80000000
-#else
+
+#else // NTOS_MODE_USER
+
 #define SEC_BASED                                           0x200000
 
 //
-// Section Inherit Flags for NtCreateSection
+// Section Inherit Flags for NtMapViewOfSection and related
```
Comment (HBelusca) [inline sdk/include/ndk/mmtypes.h:110]: "and related": are there other exported functions that also use the SECTION_INHERIT?

Comment (SergeGautherie) [reply sdk/include/ndk/mmtypes.h]: No, removed.

File: sdk/include/ndk/mmtypes.h (hunk 1 above)
Comment (SergeGautherie) [reply sdk/include/ndk/mmtypes.h:211]: Long block: 1 matching line.
Short block: 0 matching line.

Comment (HBelusca) [reply sdk/include/ndk/mmtypes.h:211]: This doesn't make much sense.

**Review by HBelusca — APPROVED** [review]

**Review by binarymaster — APPROVED** [review]
